test/e2e/e2e_test.go

```diff
@@ -246,7 +248,19 @@ func setupSharedOIDCProvider() error {
 	}
 	iamOptions.ParseAdditionalTags()
 
-	if _, err := iamOptions.CreateOIDCProvider(iamClient); err != nil {
+	createLogFile := filepath.Join(artifactDir, "create-oidc-provider.log")
```

Comment:
How would the flow look like for dev changing cmd code and running e2e locally?

Reply:
You would need to set `$ARTIFACT_DIR`, then these would show up there. I agree that streaming in that case would be nice, I think we can add that in a follow-up by adding `os.Stdout` as a sink when `$EVENTUALLY_VERBOSE!="false"`.